internal/cli/usercmd/poll.go

```diff
@@ -0,0 +1,48 @@
+// Copyright © 2021 - 2023 SUSE LLC
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//     http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package usercmd
+
+import "time"
+
+func (c *EpinioClient) trackDeletion(names []string, poller func() []string) {
+
+	// set of tracked names
+	tracked := map[string]bool{}
+	for _, name := range names {
+		tracked[name] = true
+	}
```

Comment:
I hate the lack of Set in Go. :upside_down_face:

NITP: if you don't need the boolean value you can use an empty struct `struct{}` instead of the `bool`.

Reply:
Will do. Had forgotten how write this empty struct.

Reply:
Done, commit 0d2e4c6b662550ca809b1d48cdf7101c67b508b2